app/assets/stylesheets/hyku_addons/dashboard.scss

```diff
@@ -1,3 +1,82 @@
+// Add the boostrap 4 .col-xl- breakpoint to the dashboard as version 3 only has .col-lg
```

Comment:
If we are on version 3 of Bootstrap, is there a reason why we need Bootstrap 4 css classes for this?

Reply:
Its only the `xl` grid styles, which are actually taken from Bootstrap 4, because bootstrap 3 doesn't support any breakpoints above 1200, which is ridiculous in 2022. So we now have the ability to customise the layout just a little bit more for screens over 1600 and everything else falls back to the defaults.

Reply:
Ok thanks.